sys-apps/pacman/metadata.xml

```diff
@@ -1,7 +1,21 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-  <use>
-    <flag name="gpg">Support verifying packages signed with GPG using <pkg>app-crypt/gpgme</pkg></flag>
-  </use>
+	<upstream>
+		<bugs-to>mailto:pacman-dev@archlinux.org</bugs-to>
+		<changelog>https://git.archlinux.org/pacman.git/tree/NEWS</changelog>
+		<doc>https://www.archlinux.org/pacman/</doc>
+	</upstream>
+	<maintainer type="person">
+		<email>holgersson@posteo.de</email>
+		<name>Nils Freydank</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<use>
+		<flag name="doc">Install extended documentation using <pkg>app-doc/doxygen</pkg>. (Man pages are included by default.)</flag>
```

Comment:
I think it is always better to make sure users understand what they enable by the flag. So I agree with this.